sdk/core/core-client/src/serviceClient.ts

```diff
@@ -411,7 +415,7 @@ export interface ClientPipelineOptions extends InternalPipelineOptions {
   /**
    * Options to customize bearerTokenAuthenticationPolicy.
    */
-  credentialOptions?: { baseUri?: string; credential?: TokenCredential };
+  credentialOptions?: { scope?: string; credential?: TokenCredential };
```

Comment:
Maybe we should use a consistent name here and in `ServiceClientOptions`?  `authScope` would be fine for both.

Reply:
I'm looking at autorest now and it nows this as `credential-scope` does `credentialScope` sound good to you guys? @daviwil 
, @xirzec

Reply:
`credentialScope` works for me

Reply:
`credentialScope` sounds great, I guess it's a little redundant in this case, but consistency may outweigh redundancy in this case